specs-go/config.go

```diff
@@ -529,7 +529,19 @@ type LinuxSeccompArg struct {
 
 // LinuxSyscall is used to match a syscall in Seccomp
 type LinuxSyscall struct {
-	Name   string             `json:"name"`
-	Action LinuxSeccompAction `json:"action"`
-	Args   []LinuxSeccompArg  `json:"args,omitempty"`
```

Comment:
You seem to be replacing `LinuxSeccompArg` with `LinuxSyscallArg`, but that leaves a dangling `LinuxSeccompArg` type (at least as of c36c819):

```
$ git grep LinuxSeccompArg origin/pr/657
origin/pr/657:specs-go/config.go:// LinuxSeccompArg used for matching specific syscall arguments in Seccomp
origin/pr/657:specs-go/config.go:type LinuxSeccompArg struct {
```

Also, we'll need to update the Markdown spec to cover this change, since there is definitely a schema change going on and the Go types are not normative.  The JSON Schema will need updating as well.

```
$ git show --stat --oneline origin/pr/657
c36c819 improve seccomp format to be more expressive
 specs-go/config.go | 32 ++++++++++++++++++++++----------
 1 file changed, 22 insertions(+), 10 deletions(-)
```